app/src/main/java/co/develhope/chooseyourownbeer/MainActivity.kt

```diff
@@ -34,4 +58,18 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
     }
+    fun retrievePunkRepos(){
+        lifecycleScope.launch{
+            try{
+                val repos = punkService.listRepos("beers")
```

Comment:
Eliminerei il passaggio del parametro credo non serva, ma se dovesse servire e non cambia mai andrebbe cmq inserito in una costante in modo da usare sempre lo stesso valore

Reply:
Ho inserito questo parametro nella funzione retriveBeerRepo() nel MainViewModel dimmi se va bene